### DISCUSSION TITLE
Is there a check for formal speech?

### DISCUSSION THREAD

**Discussion by JulianGro** [WeblateOrg/weblate#7458] [Q&A] [answered]
Is there a check for formal speech?
In some languages, like German, formal speech is starting to become less important. Translators now tend to mix formal and informal speech.
I would like to check that warns about informal speech.
Even just checking for a hand full of informal words would be of great help.

**Comment by nijel** [comment, marked as answer]
There is currently no such check.

If you know some library that would allow implementing it, please open a feature request.
